lib/cloudant.js

```diff
@@ -770,19 +515,19 @@ Cloudant.prototype.updateOrCreate = function(model, data, cb) {
  * @param {String} model The model name
  * @param {Object} where The search criteria
  * @param {Object} data The property/value pairs to be updated
- * @callback {Function} cb Cb function
+ * @param {Object} options The options Object
+ * @param {Function} [cb] The callback function
```

Comment:
Hmm, seems the right syntax is `@callback {Function} callback` according to current juggler code.